smoketest/src/main/java/com/mongodb/jdbc/smoketest/SmokeTest.java

```diff
@@ -0,0 +1,70 @@
+package com.mongodb.jdbc.smoketest;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+
+/**
+ * SmokeTest runs a test on built artifacts to verify that connection,
+ * metadata retrieval, and querying is successful
+ */
+public class SmokeTest {
```

Comment:
Why make this an executable with a `main` instead of a junit test with two test methods?

Reply:
That's a good idea, I switched it to junit tests